src/contracts/vendored/mixins/BridgedTokensRegistry.sol

```diff
@@ -10,35 +10,60 @@ import "./EternalStorage.sol";
  * @dev Functionality for keeping track of registered bridged token pairs.
  */
 contract BridgedTokensRegistry is EternalStorage {
-    event NewTokenRegistered(address indexed nativeToken, address indexed bridgedToken);
+    event NewTokenRegistered(
```

Comment:
I don't understand it. On the URL, it looks like this: 
![image](https://user-images.githubusercontent.com/7348154/150973007-54a8f737-4b63-405d-b2a8-f0f308c4dad0.png)
Maybe there is some under the hood formatting going on?

Reply:
Yes! Note that the line running the diff is:
$ diff <(curl --silent "${vendored_url:1:-1}" | **npx prettier --parser solidity-parse**) "$contract"

This is because I noticed that it was easier to format everything than to try to be careful at never autoformatting anything. Also, there would be no question on how to format new changes.